### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by CindyDorantes**
## Project Requirements   :bookmark_tabs:

Use [this template](https://github.com/microverseinc/curriculum-template-databases) to generate your repo.
- Create a database named `vet_clinic`.
- Create a table named `animals` with the following columns:
  - `id`: integer
  - `name`: string
  - `date_of_birth`: date
  - `escape_attempts`: integer
  - `neutered`: boolean
  - `weight_kg`: decimal
- Copy the SQL you used in the previous point into the `schema.sql` file in the generated repository.
- Insert the data provided.
- Copy the SQL you used in the previous point into the `data.sql` file in the generated repository.
- Write queries for the provided quests
- **Take a screenshot of the results of your queries.**
- Copy the SQL you used in the previous point into the `queries.sql` file in the generated repository.

## Screenshots  :camera:

query_animals_01

query_animals_02

query_animals_03

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals ( name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES 
+('Agumon','2020-02-03',0,TRUE,10.23),
+('Gabumon','2018-11-15',2,TRUE,8.0),
+('Pikachu','2021-01-07',1,FALSE,15.04),
+('Devimon','2017,05,12',1,FALSE,15.04);
+
```
Comment (RolandM99) [inline data.sql:8]: - [x] The weight for the animal with the name `Devimon` should be `11.0` , the `escape_attempts` is also `'5'` and the `neutered` is on `'true'`. Kindly, correct your queries.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi  @CindyDorantes , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Gitflow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (CindyDorantes) [reply data.sql:8]: Thanks for the observation, I have updated the values :wink:

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @CindyDorantes ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**